index.html

```diff
@@ -8438,6 +8460,7 @@ <h5>Presentational Roles Conflict Resolution</h5>
 			<div class="role-description">
 				<p>A <rref>section</rref> containing data arranged in rows and columns. See related <rref>grid</rref>.</p>
 				<p>The <code>table</code> role is intended for tabular containers which are not interactive. If the tabular container maintains a selection state, provides its own two-dimensional navigation, or allows the user to rearrange or otherwise manipulate its contents or the display thereof, authors SHOULD use <rref>grid</rref> or <rref>treegrid</rref> instead.</p>
+				<p>Authors MUST give each element with role table a brief label that describes the purpose of the content in the table.  Authors MAY use a visible caption as the first child element of the table to provide a label.  The caption MAY be an instance of the standard host language <code>caption</code> element or an instance of an element with role <rref>caption</rref>.  If a visible label is present, but not contained in the first child element with a role of <code>caption</code>, authors SHOULD reference the visible label with <pref>aria-labelledby</pref>.</p>
```

Comment:
change:
>but not contained in the first child element with a role of <code>caption</code>

to:
>but not the first child element of an element with a role of <code>table</code>

and again, mention `aria-owns` as well as labelling?

Reply:
@scottaohara made the change to "but not the first child element of an element with a role of `table`"